hack/build/setup-pac-integration.sh

```diff
@@ -85,4 +85,5 @@ if ! oc get namespace ${PAC_NAMESPACE} &>/dev/null; then
 fi
 oc -n ${PAC_NAMESPACE} delete secret ${PAC_SECRET_NAME} &>/dev/null
 eval "oc -n '$PAC_NAMESPACE' create secret generic '$PAC_SECRET_NAME' $GITHUB_APP_DATA $GITHUB_WEBHOOK_DATA $GITLAB_WEBHOOK_DATA"
+eval "oc -n build-service create secret generic '$PAC_SECRET_NAME' $GITHUB_APP_DATA $GITHUB_WEBHOOK_DATA $GITLAB_WEBHOOK_DATA"
```

Comment:
The script fails here, because build-service namepsace is not created yet:
```
...
namespace/pipelines-as-code created
secret/pipelines-as-code-secret created
error: failed to create secret namespaces "build-service" not found
Configured pipelines-as-code-secret secret in pipelines-as-code namespace
...
```

Reply:
That might be the cause of the PaC Pipelineruns not being started, no? 
I can see these errors in build service controller manager:
```1.6686931783899896e+09	DEBUG	events	Warning	{"object": {"kind":"Secret","apiVersion":"v1"}, "reason": "ErrorReadingPaCSecret", "message": "secrets \"pipelines-as-code-secret\" not found"}
1.6686932988572547e+09	INFO	controllers.ComponentOnboarding	Pipelines as Code enabled	{"ComponentOnboarding": "build-e2e-obpg/test-component-pac-zobf"}
1.668693298863861e+09	ERROR	controllers.ComponentOnboarding	failed to get Pipelines as Code secret in build-service namespace	{"ComponentOnboarding": "build-e2e-obpg/test-component-pac-zobf", "error": "secrets \"pipelines-as-code-secret\" not found"}
```
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/redhat-appstudio_infra-deployments/927/pull-ci-redhat-appstudio-infra-deployments-pre-kcp-appstudio-e2e-deployment/1593238191152828416/artifacts/appstudio-e2e-deployment/gather-extra/artifacts/pods/build-service_build-service-controller-manager-7cd9bcc574-8tbw2_manager.log